src/dodal/devices/xspress3_mini/xspress3_mini_channel.py

```diff
@@ -1,6 +1,14 @@
+from enum import Enum
+
 from ophyd import Component, Device, EpicsSignal, EpicsSignalRO
 
 
+class TimeSeriesValues(Enum):  # assuming IOC version 3
```

Comment:
Should: I think drop this comment, I'm 90% sure we're on all the same IOC version now